### PR TITLE
[WIP] Hybrid ui integration

### PR DESCRIPTION
**Pull request by bdunogier**
Integrates [ezsystems/hybrid-platform-ui](http://github.com/ezsystems/hybrid-platform-ui) into ezplatform.

### CODE REVIEW

**Comment by andrerom** [comment]
Add this on 2.0 branch?

**Comment by bdunogier** [comment]
> Add this on 2.0 branch?

Why not. It will require platform-ui 2.0 anyway.

File: composer.json [hunk 1]
```diff
@@ -14,13 +14,19 @@
         "ezsystems/ezpublish-community": "*"
     },
     "autoload": {
+<<<<<<< HEAD
```
Comment (dpobel) [inline composer.json]: oops :p

File: composer.json [hunk 2]
```diff
@@ -44,8 +50,14 @@
...
         "doctrine/doctrine-bundle": "~1.5",
         "sensio/framework-extra-bundle": "~3.0",
-        "ezsystems/content-on-the-fly-prototype": "~0.1.7",
         "willdurand/js-translation-bundle": "^2.6.4"
+
+        ,"friendsofsymfony/jsrouting-bundle": "^1.6",
+        "composer/installers": "dev-ezplatform_assets as 1.3.x-dev",
+        "dpobel/ez-field-edit": "dev-master as 1.0.x-dev",
+        "dpobel/ez-map": "dev-master as 1.0.x-dev",
+        "dpobel/hybrid-platformui-assets": "~0.3",
```
Comment (dpobel) [inline composer.json]: I've created and published version 0.1.0 of ezsystems/hybrid-platform-ui-assets which can replace this one. That also means this dependency can (and should) be moved tohybrid-platform-ui composer.json.

File: composer.json [hunk 3]
```diff
@@ -44,8 +50,14 @@
         "hautelook/templated-uri-bundle": "~2.0",
         "doctrine/doctrine-bundle": "~1.5",
         "sensio/framework-extra-bundle": "~3.0",
-        "ezsystems/content-on-the-fly-prototype": "~0.1.7",
         "willdurand/js-translation-bundle": "^2.6.4"
+
+        ,"friendsofsymfony/jsrouting-bundle": "^1.6",
+        "composer/installers": "dev-ezplatform_assets as 1.3.x-dev",
+        "dpobel/ez-field-edit": "dev-master as 1.0.x-dev",
```
Comment (dpobel) [inline composer.json]: given content edit is not supposed to work yet, I would remove dpobel/ez-field-edit and dpobel/ez-map

File: composer.json [hunk 4]
```diff
@@ -44,8 +50,14 @@
         "hautelook/templated-uri-bundle": "~2.0",
         "doctrine/doctrine-bundle": "~1.5",
         "sensio/framework-extra-bundle": "~3.0",
-        "ezsystems/content-on-the-fly-prototype": "~0.1.7",
         "willdurand/js-translation-bundle": "^2.6.4"
+
+        ,"friendsofsymfony/jsrouting-bundle": "^1.6",
```
Comment (dpobel) [inline composer.json]: this one should rather be in PlatformUIBundle composer.json (and most likely at some point in   ezsystems/hybrid-platform-ui as well)

Comment (bdunogier) [inline composer.json]: This one can indeed be moved. It needs to be in hybrid-platform-ui, as the layout.html.twig includes a link to it. I'll add it.

File: composer.json (hunk 3 above)
Comment (bdunogier) [inline composer.json]: Don't be sad, @andrerom, we are just keeping it manageable. The prototypes are still around :)

File: composer.json (hunk 2 above)
Comment (bdunogier) [inline composer.json]: Great, I'll clean that up too.

File: composer.json (hunk 3 above)
Comment (andrerom) [inline composer.json]: I know. And it's not a sad face, it's I want an ice cream face, but I understand it's to cold outside today so I can wait :)

**Comment by bdunogier** [comment]
Should be all cleaned up and installable.

File: app/config/config.yml [hunk 5]
```diff
@@ -67,8 +67,11 @@ twig:
 
 # Assetic Configuration
 assetic:
-    debug: '%kernel.debug%'
+    debug:          '%kernel.debug%'
```
Comment (andrerom) [inline app/config/config.yml]: can skip this :)

File: app/config/config.yml [hunk 6]
```diff
@@ -67,8 +67,11 @@ twig:
...
     use_controller: '%kernel.debug%'
+    use_controller: "%kernel.debug%"
```
Comment (andrerom) [inline app/config/config.yml]: duplicate

File: composer.json [hunk 7]
```diff
@@ -47,6 +47,9 @@
         "sensio/framework-extra-bundle": "^3.0.2",
         "ezsystems/content-on-the-fly-prototype": "^0.1.7",
         "willdurand/js-translation-bundle": "^2.6.4"
+        "composer/installers": "dev-ezplatform_assets as 1.3.x-dev",
```
Comment (andrerom) [inline composer.json]: something strange here it seems

**Comment by andrerom** [comment]
But maybe rebase away the fixup commits when merging with merge commit _(to be able to revert the disabling of multi upload & cotf later)_

File: composer.json (hunk 7 above)
Comment (bdunogier) [inline composer.json]: `composer validate` is okay with it. Tried to replace it with a fresh `:`, no diff. We'll see :)

**Comment by bdunogier** [comment]
And merged to 2.0.

**Pull request merged**